rust/kernel/lib.rs

```diff
@@ -13,11 +13,16 @@
 
 #![no_std]
 #![feature(arbitrary_self_types)]
-#![feature(coerce_unsized)]
-#![feature(dispatch_from_dyn)]
+#![cfg_attr(
+    CONFIG_RUST_COERCE_POINTEE,
+    feature(derive_coerce_pointee, pin_coerce_unsized_trait)
+)]
+#![cfg_attr(
+    not(CONFIG_RUST_COERCE_POINTEE),
+    feature(coerce_unsized, dispatch_from_dyn, unsize)
+)]
```

Comment:
Does this format better if we split each feature out on its own line?

Reply:
I have splitted them in the final patch. I will push the results shortly.